File: src/main/java/com/vimeo/networking/model/UserList.java
package com.vimeo.networking.model;

import java.util.ArrayList;

/**
 * Created by hanssena on 4/23/15.
 */
public class UserList
{
    public int total;
    public int page;
    public int perPage;
    public Paging paging;
    public ArrayList<User> data;
}

File: models/src/main/java/com/vimeo/networking2/UserList.kt
package com.vimeo.networking2

import com.vimeo.networking2.common.Pageable

/**
 * List of users that could be paged.
 */
data class UserList(
    override val total: Int? = null,
    override val page: Int? = null,
    override val perPage: Int? = null,
    override val paging: Paging? = null,
    override val data: List<User>? = null
) : Pageable<User>
